content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -117,18 +118,16 @@ Docker EE-basic 19.03+ is the recommended container runtime for all Windows Serv
 
 ##### CRI-ContainerD
 
-{{< feature-state for_k8s_version="v1.19" state="beta" >}}
+{{< feature-state for_k8s_version="v1.20" state="stable" >}}
 
-{{< caution >}}
-There is a [known limitation](/docs/tasks/configure-pod-container/configure-gmsa/#gmsa-limitations) when using GMSA with ContainerD to access Windows network shares which requires a kernel patch. Check for updates on the [Microsoft Windows Containers issue tracker](https://github.com/microsoft/Windows-Containers/issues/44).
-{{< /caution >}}
-
-{{< glossary_tooltip term_id="containerd" text="ContainerD" >}} 1.4.0-beta.2+ can also be used as the container runtime for Windows Kubernetes nodes.
-
-Initial support for ContainerD on Windows was added in Kubernetes v1.18. Progress for ContainerD on Windows can be tracked at [enhancements#1001](https://github.com/kubernetes/enhancements/issues/1001).
+{{< glossary_tooltip term_id="containerd" text="ContainerD" >}} 1.4.0+ can also be used as the container runtime for Windows Kubernetes nodes.
 
 Learn how to [install ContainerD on a Windows](/docs/setup/production-environment/container-runtimes/#install-containerd).
 
+{{< caution >}}
+There is a [known limitation](/docs/tasks/configure-pod-container/configure-gmsa/#gmsa-limitations) when using GMSA with ContainerD to access Windows network shares which requires a kernel patch. Updates to address this limitation are expected to be available for Windows Server 2019 and later in the November 2020 optional update package (11C). Check for updates on the [Microsoft Windows Containers issue tracker](https://github.com/microsoft/Windows-Containers/issues/44).
```

Comment:
Need to confirm with @immuzz that GMSA support for Windows Server 2019 will be available in Nov 2020 (and if not update this section)

Reply:
@marosset  were you able to clarify this?

Reply:
Yes, looks like it will be available early 2021. Let me reopen that issue and update this caution section.

Reply:
Bear in mind that our default setting is to [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future) entirely. It's OK _not_ to make a date commitment.